manifests/cluster-manager/cluster-manager-registration-clusterrole.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: system:open-cluster-management:{{ .ClusterManagerName }}-registration-controller
```

Comment:
i think  the format is something like open-cluster-management:registration:{{ .ClusterManagerName }}-controller ?

Reply:
done